core/domain/exp_jobs_one_off.py

```diff
@@ -678,10 +683,46 @@ def map(file_model):
                             content, mimetype='image/%s' % filetype)
                         yield (FILE_COPIED, 1)
 
-
     @staticmethod
     def reduce(status, values):
         if status == FILE_COPIED:
             yield (status, len(values))
         else:
             yield (status, values)
+
+
+class CreateVersionsOfImageJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for creating compressed versions of the images
+    of the exploration in the GCS.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(exp_model):
+        exploration = exp_services.get_exploration_by_id(exp_model.id)
+        current_version = exploration.version
+        print current_version
```

Comment:
Drop this. (@apb7, we should forbid print statements in the linter.)

Reply:
Done